rake/spec/default/70-st2_actions-integrity_spec.rb

```diff
@@ -6,4 +6,12 @@
   describe command("st2 run packs.install packs=github") do
     its(:exit_status) { is_expected.to eq 0 }
   end
+
+  describe command("st2 run core.local cmd='locale'") do
+    its(:stdout) { should match /UTF-8/ }
```

Comment:
nice one :+1: 

@userlocalhost If Serverspec doesn't work, I'm OK with only this test.